config/Dockerfiles/create_pypi_package.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
```

Comment:
What is the reason for moving the package from script to config/Dockerfiles?

Reply:
Also - are we not using this file anymore?  I see everything is mostly commented out.  is the twine command doing the upload now?  Just trying to make sure I understand it all. :-)

Reply:
Yeah. I just need to remove this file. I'll update that shortly, in addition to a couple other cleanups mentioned above.